packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js

```diff
@@ -46,6 +46,56 @@ export const transforms = toArray('inbound', {
     return ctx.transform(`decrypt${capitalize(object.objectType)}`, key, object);
   },
 
+
+  /**
+   * Decrypt an individual reaction2Summary activity (object.objectType === 'reaction2Summary')
+   * @param   {Object} ctx      An object containg a webex instance and a transform
+   * @param   {String} key      KMS encryption key url (or actual key?)
+   * @param   {Object} object An instance of a Webex Teams reaction2Summary object
+   * these objects are returned by various conversation APIs and over mercury
+   * represents an aggregated summary of all reactions to a specific activity
+   * @returns {Promise}         Returns a ctx.transform promise
+   */
+  decryptReaction2summary(ctx, key, object) {
+    if (!object.reactions) {
+      return Promise.resolve();
+    }
+
+    return Promise.all(object.reactions.map((reaction) => ctx.transform('decryptPropDisplayName', key, reaction)));
+  },
+
+  /**
+   * Decrypt an individual reaction2SelfSummary activity (object.objectType === 'reaction2SelfSummary')
+   * @param   {Object} ctx      An object containg a webex instance and a transform
+   * @param   {String} key      KMS encryption key url (or actual key?)
+   * @param   {Object} object An instance of a Webex Teams reaction2SelfSummary object
+   * these objects are returned by various conversation APIs and NOT over mercury
+   * they are ONLY received by the self user
+   * they represent ONLY the self user's reactions and are used for enforcing
+   * limit of times they can react to a specific activity
+   * @returns {Promise}         Returns a ctx.transform promise
+   */
+  decryptReaction2selfsummary(ctx, key, object) {
+    if (!object.reactions) {
+      return Promise.resolve();
+    }
+
+    return Promise.all(object.reactions.map((reaction) => ctx.transform('decryptPropDisplayName', key, reaction)));
+  },
+
+  /**
+   * Decrypt an individual reaction2 activity (object.objectType === 'reaction2')
+   * @param   {Object} ctx      An object containg a webex instance and a transform
+   * @param   {String} key      KMS encryption key url (or actual key?)
+   * @param   {Object} object An instance of a Webex Teams reaction2 object
+   * these objects are returned by various conversation APIs and over mercury
+   * ONLY self users receive these objects
+   * @returns {Promise}         Returns a ctx.transform promise
+   */
+  decryptReaction2(ctx, key, object) {
+    return ctx.transform('decryptPropDisplayName', key, object);
```

Comment:
i'm happy to if we'd like. both of these methods get called from the `return ctx.transform(`decrypt${capitalize(object.objectType)}`, key, object);` in `decryptObject`. the main reasons i didn't do that are because:
1 - the bodies of these methods are very small and simple.
2 - i could potentially see these object shapes changing in the future, so it could be helpful to have them separated based on `objectType`
3 - as far as i can there aren't any other sort of "helper/util" functions in this file. it seems to be 1:1 relationship of each objectType has it's own decrypt function, and i thought it might be simpler to continue that pattern.

Reply:
yeah, it make sense now that I understand better the design of this module (not that I necessarily agree with the it but 🤷🏻‍♀️).
At first glance, it was just weird to see methods doing almost identical things over and over.